doc-content/enterprise-only/release-notes/rn-770-fixed-issues-ref.adoc

```diff
@@ -0,0 +1,72 @@
+[id='rn-770-fixed-issues-ref']
+= Fixed issues in {PRODUCT} 7.7.0
+
+{PRODUCT} 7.7.0 provides increased stability and fixed issues listed in this section.
+
+ifdef::PAM[]
+[NOTE]
+====
+{PRODUCT_DM} fixed issues apply to {PRODUCT_PAM} as well. For a list of {PRODUCT_DM} 7.7.0 fixed issues, see the https://access.redhat.com/documentation/en-us/red_hat_decision_manager/7.7/html-single/release_notes_for_red_hat_decision_manager_7.6/index[_Release Notes for Red Hat Decision Manager 7.7_]
+====
+
+== {CENTRAL}
+
+* When you add a deployment unit and manually enter GAV values, the runtime strategy is not set to the configured default value [https://issues.redhat.com/browse/RHPAM-2623[RHPAM-2623]]
+* When you save a filter and then activate it, an error message displays [https://issues.redhat.com/browse/RHPAM-2545[RHPAM-2545]]
+* In the guided rule editor, you cannot use the combination of complex values [https://issues.redhat.com/browse/RHPAM-2457[RHPAM-2457]]
+* The state of a {KIE_SERVER} is not updated in the server template after the server disconnects and reconnects to {CENTRAL} [https://issues.redhat.com/browse/RHPAM-2544[RHPAM-2544]]
+* Guided rule editor removes `matches` operator from rule file [https://issues.redhat.com/browse/RHPAM-2631[RHPAM-2631]]
```

Comment:
The guided rule editor removes the `matches`... 

...from rule fire
Well, this is a little unclear. What is a rule fire? And is it a rule fire or the rule fire?